src/routes/plans/[id]/+page.svelte

```diff
@@ -149,7 +149,30 @@
       removeQueryParam(SearchParameters.ACTIVITY_ID);
     }
 
-    $viewTimeRange = $maxTimeRange;
+    let start = NaN;
+    const startTimeStr = $page.url.searchParams.get(SearchParameters.START_TIME);
```

Comment:
Wonder if we should be using the `getSearchParameterNumber` generic util function around here - looks like @duranb has used it recently and might know more?

Reply:
Despite my preference below that that we clear the query parameters, I did want this expressed in ISO format in the query string to make it easier for constructing a custom url